rmw/include/rmw/types.h

```diff
@@ -326,7 +332,12 @@ typedef struct RMW_PUBLIC_TYPE rmw_time_t
 
   /// Nanoseconds component of this time point
   uint64_t nsec;
-} rmw_time_t;
+} RMW_DECLARE_DEPRECATED (rmw_time_t,
+  "rmw_time_t has been deprecated in G-Turtle in favor of rmw_duration_t. "
```

Comment:
minor suggestion; might be a bit more clear
```suggestion
  "rmw_time_t has been deprecated in Galactic in favor of rmw_duration_t. "
```